packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -44,20 +45,30 @@ class Series extends Component {
   componentDidUpdate (prevProps) {
     const { visible, data, ...rest } = this.props;
 
+    let needsRedraw = false;
     // Using setData is more performant than update
     if (isImmutable(data) && immutableEqual(data, prevProps.data) === false) {
-      this.series.setData(data.toJS(), true);
+      this.series.setData(data.toJS(), false);
+      needsRedraw = true;
     } else if (isEqual(data, prevProps.data) === false) {
-      this.series.setData(data, true);
+      this.series.setData(data, false);
+      needsRedraw = true;
     }
     if (visible !== prevProps.visible) {
-      this.series.setVisible(visible);
+      this.series.setVisible(visible, false);
+      needsRedraw = true;
     }
 
     const modifiedProps = getModifiedProps(prevProps, rest);
     if (modifiedProps !== false) {
-      this.series.update(modifiedProps);
+      this.series.update(modifiedProps, false);
```

Comment:
Think we need a `needsRedraw = true` after this line?

Reply:
Oops..